templates/collections/explorer_index_page.html

```diff
@@ -7,26 +7,28 @@
 {% block content %}
     {% include "includes/generic-intro.html" with title=page.title intro=page.intro %}
 
-    <div class="container mt-4" id="analytics-explorer-index" data-container-name="explorer-index">
-       <div class="row"> <!-- row for large link cards -->
-            <ul class="card-group--no-flex">
-                {% for block in page.body %}
-                    {% include_block block %}
-                {% endfor %}
-            </ul>
-        </div>
+    <div id="analytics-explorer-index" data-container-name="explorer-index">
+        <ul class="card-group--no-flex">
+            {% for block in page.body %}
+                {% include_block block %}
+            {% endfor %}
+        </ul>
+    </div>
 
-        <h2>{{ page.articles_title }}</h3> {% comment %} TODO: Not sure about the choice of h2 and p, something to pick up in the FE work {% endcomment %}
+    <div class="container">
+        <h2 class="extra-padding">{{ page.articles_title }}</h3>
 
         <p>{{ page.articles_introduction }}</p>
-        {% if page.featured_article %}
-            {% include "includes/article-spotlight.html" with page=page.featured_article.specific %}
-        {% endif %}
-        <div class="container">
-            {% include_block page.featured_articles %}
-        </div>
     </div>
 
+    {% if page.featured_article %}
+        {% include "includes/article-spotlight.html" with page=page.featured_article.specific %}
+    {% endif %}
+
+    <div class="container">
+        {% include_block page.featured_articles %}
+    </div>
+    
```

Comment:
I think there should also be a link to the stories index page at the bottom here:

![image](https://github.com/nationalarchives/ds-wagtail/assets/62654785/6662ffb9-98d6-445f-a4d4-729e9189b39b)